src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -194,24 +227,25 @@ export default class PickEntityOrConversionDialog extends CommonMixin {
     return output
   }
 
-  get width (): string {
-    if (this.showSocietiesInfo || this.isConversion) {
+  get maxWidth (): string {
+    if (this.showSocietiesInfo || this.isConversion || this.isAmalgamation) {
       return '550px'
     }
-    let cols = this.tableData.length
-    const maxThreshold = this.$vuetify.breakpoint.thresholds.sm
 
-    // 210 per column with a max threshold of 960px
-    return `${210 * cols > maxThreshold ? maxThreshold : 210 * cols}px`
+    // 210 per column with a max threshold of 960px (sm)
+    const cols = this.tableData.length
+    const maxThreshold = this.$vuetify.breakpoint.thresholds.sm
+    const val = (210 * cols > maxThreshold) ? maxThreshold : (210 * cols)
+    return `${val}px`
```

Comment:
Just cleanup.

Reply:
When you added the `.sm`, you're targeting the threshold for small screens?